Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2585 @@
+    elif query_type == 'event':
+        search_query = get_event_filter_query(args)
+    elif query_type == 'incident':
+        search_query = get_incident_filter_query(args)
+    else:
+        # default
+        search_query = args.get('query', '')
+
+    if search_query:
+        payload['query'] = search_query
+
+    return payload
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def common_wrapper_command(client_func: Callable, cmd_args: dict, readable_title: str,
+                           context_path: str, output_key_field: str, command_type: str = None,
+                           is_call_diff_readable_output: bool = False,
+                           func_readable_output: Callable = generic_readable_output, **kwargs) -> CommandResults:
+    """
+    Common Wrapper Command for different endpoints
+    Args:
+        client_func: Call client method e.g. client.list_domain_file
+        cmd_args: Command arguments, usually passed from ``demisto.args()``.
+        readable_title: Readable Output title
+        context_path: Readable Context Output path
+        output_key_field: Outputs key field
+        command_type: Load the specific payload
+        is_call_diff_readable_output: Default False, Only pass the Ture in case need to call different
+                    readable output method
+        func_readable_output: Optional, call in case of readable output method is different for specific command
+        kwargs: In case required other arguments
+
+    Returns:
+        CommandResults: A ``CommandResults`` object
+    """
+    context_data: list = []
+    payload = create_payload_for_query(cmd_args, command_type)
+    offset = int(payload.pop('offset', ''))
+    limit = int(payload.get('limit', ''))
+
+    if 'uuid' in kwargs:
+        raw_response = client_func(payload, kwargs['uuid'])
+    elif 'sha2' in kwargs:
+        raw_response = client_func(payload, kwargs['sha2'])
+    else:
+        raw_response = client_func(payload)
+
+    title = compile_command_title_string(readable_title, cmd_args, int(raw_response.get('total', 0)))
+
+    if printable_result := get_data_of_current_page(raw_response.get('result', []), offset, limit):
+        if is_call_diff_readable_output:
+            if 'incident_id' in kwargs:
+                readable_output, context_data = func_readable_output(printable_result, title, kwargs['incident_id'])
+            else:
+                readable_output, context_data = func_readable_output(printable_result, title)
+        else:
+            readable_output = generic_readable_output(printable_result, title)
+            context_data = printable_result
+    else:
+        readable_output = f'No {readable_title} data to present.'
+
+    return CommandResults(
+        outputs_prefix=f'{INTEGRATION_CONTEXT_NAME}.{context_path}',
+        outputs_key_field=output_key_field,
+        readable_output=readable_output,
+        outputs=context_data,
+        raw_response=raw_response,
+        ignore_auto_extract=True
```

Comment:
why `ignore_auto_extract`?

Reply:
This was suggested during the demo.

Reply:
@DanilVil should we use it here?

Reply:
@snath-loginsoft @dorschw We have requested this after the demo for the result returned for ׳symantec-edr-endpoint-isolate׳ command and all other endpoint-related commands - This means, only commands that involve actual endpoints.

Reply:
@DanilVil, @dorschw, just verified and it set as True for all the endpoint related commands. I am assuming that it will be applicable for all the commands irrespective of endpoints. 

I will go and remove " ignore_auto_extract=True" from other commands.